### PR TITLE
Preview Card Unit Test

### PR DESCRIPTION
**Pull request by JeffersonMat**
Link to associated zer0 card:  

-----

## 1. Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] zer0 card has been moved to the Code Review column
- [x] zer0 card has a link to this PR
- [x] A reviewer has been tagged in the zer0 card

## 2. PR type

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## 3. What is the current behavior?

PreviewCard component doesn't have unit test
## 4. What is the new behavior?

Add unit test for previewCard component

### CODE REVIEW

**Comment by ericlowturing** [comment]
LGTM! Overall is very good. I just left minor comments with change requests. Approving by pending those comments

**Comment by JeffersonMat** [comment]
> LGTM! Overall is very good. I just left minor comments with change requests. Approving by pending those comments

@ericlowturing Hey thanks! I don't seem to find the comments. Could you please let me know where I can find them?

File: src/components/Cards/PreviewCard/PreviewCard.tsx [hunk 1]
```diff
@@ -27,6 +27,21 @@ type PreviewCardProps = {
 	style?: React.CSSProperties;
 };
 
+export const TEST_ID_PARENT = {
```
[the remark this answers is not in the document]
Comment (ericlowturing) [reply src/components/Cards/PreviewCard/PreviewCard.tsx:30]: @JeffersonMat  Can you see my comment above?

File: src/components/Cards/PreviewCard/PreviewCard.tsx [hunk 2]
```diff
@@ -107,19 +129,30 @@ const PreviewCard: React.FC<PreviewCardProps> = ({
 	////////////
 
 	return (
-		<div className={styles.PreviewCard} style={style ? style : {}}>
-			{preventInteraction && <div className={styles.Blocker}></div>}
+		<div
+			className={styles.PreviewCard}
+			style={style ? style : {}}
```
[the remark this answers is not in the document]
Comment (ericlowturing) [reply src/components/Cards/PreviewCard/PreviewCard.tsx:134]: @JeffersonMat Can you see my comment above?

File: src/components/Cards/PreviewCard/Previewcard.test.tsx [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+			domain={domain}
+			image={image}
+			isLoading={isLoading}
+			mvpVersion={mvpVersion}
+			name={name}
+			ownerId={ownerId}
+			style={style}
+		/>,
+	);
+};
```
[the remark this answers is not in the document]
Comment (ericlowturing) [reply src/components/Cards/PreviewCard/Previewcard.test.tsx:35]: @JeffersonMat Can you see my comment above?

File: src/components/Cards/PreviewCard/Previewcard.test.tsx [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+	it('Asset element Does have classname', () => {
+		const { getByTestId } = renderComponent({ mvpVersion: 3 });
+		const asset = getByTestId(TEST_ID_PARENT.ASSET);
+		expect(asset).toHaveClass('MVP3Asset');
+	});
+
+	it('Does Contain </NFTMEDIA>', () => {
+		const { getByTestId } = renderComponent();
+		const asset = getByTestId(TEST_ID_PARENT.ASSET);
+		expect(asset.childElementCount).toBe(1);
```
[the remark this answers is not in the document]
Comment (ericlowturing) [reply src/components/Cards/PreviewCard/Previewcard.test.tsx:101]: @JeffersonMat Can you see my comment above?

**Review by ericlowturing — APPROVED** [review]
LGTM! 💯

**Comment by ericlowturing** [comment]
> > LGTM! Overall is very good. I just left minor comments with change requests. Approving by pending those comments
> 
> @ericlowturing Hey thanks! I don't seem to find the comments. Could you please let me know where I can find them?

@JeffersonMat I am sorry, I left a comment in this morning, but it was pending for some reason. I just left again!

File: src/components/Cards/PreviewCard/PreviewCard.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (JeffersonMat) [reply src/components/Cards/PreviewCard/PreviewCard.tsx:30]: @ericlowturing Yeah, I thought I had updated it. I was thinking to split the test id into two. Anyway, this has been done.
Thank you.

File: src/components/Cards/PreviewCard/PreviewCard.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (JeffersonMat) [reply src/components/Cards/PreviewCard/PreviewCard.tsx:134]: @ericlowturing  Thank you. I have made the change accordingly.

File: src/components/Cards/PreviewCard/Previewcard.test.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (JeffersonMat) [reply src/components/Cards/PreviewCard/Previewcard.test.tsx:35]: @ericlowturing Nice. Thank you!

File: src/components/Cards/PreviewCard/Previewcard.test.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (JeffersonMat) [reply src/components/Cards/PreviewCard/Previewcard.test.tsx:101]: @ericlowturing  Done. This god to know.

File: src/components/Cards/PreviewCard/Previewcard.test.tsx [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+				className: 'Image img-border-rounded',
+				alt: 'NFT Preview',
+				ipfsUrl: image,
+			}),
+		);
+	});
+
+	it('Does Not Render </TextButton>', () => {
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
+		const { getByTestId } = renderComponent({ mvpVersion: 0 });
```
[the remark this answers is not in the document]
Comment (JeffersonMat) [reply src/components/Cards/PreviewCard/Previewcard.test.tsx:123]: @ericlowturing This is good to know. Thank you!

File: src/components/Cards/PreviewCard/Previewcard.test.tsx [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+		const { getByTestId } = renderComponent();
+		const asset = getByTestId(TEST_ID_PARENT.ASSET);
+		expect(asset.childElementCount).toBe(1);
+	});
+
+	it('Does Render Children NFTMedia with image url', () => {
+		const image =
+			'https://res.cloudinary.com/fact0ry/image/upload/c_…ns/QmRLG913uKX7QxwSFMk1TMhtjxwy6kVek37HTcR7AtJUVf';
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
+		const { getByTestId } = renderComponent({
```
[the remark this answers is not in the document]
Comment (JeffersonMat) [reply src/components/Cards/PreviewCard/Previewcard.test.tsx:108]: @ericlowturing Done. Thank you.

File: src/components/Cards/PreviewCard/Previewcard.test.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (ericlowturing) [reply src/components/Cards/PreviewCard/Previewcard.test.tsx]: Thanks!

**Comment by padala** [comment]
@ericlowturing @JeffersonMat  Is this good to merge?

**Comment by colbr** [comment]
@ericlowturing @JeffersonMat @padala 

Bumping this one -- how are we looking?

**Comment by ericlowturing** [comment]
> @ericlowturing @JeffersonMat @padala
> 
> Bumping this one -- how are we looking? Should try to get this PR merged ASAP to clear out the list

Yep, Its good to merge. I thought it merged already. Can you please try rebase and run `npm test` again? Or we can see CircleCi test here. Anyway, we are good to merge since it is test not a dev/feature change.

**Pull request merged**